src/lib/webpack/run-webpack.js

```diff
@@ -146,19 +151,33 @@ export function writeJsonStats(stats) {
 	});
 }
 
-const keysToNormalize = ['identifier', 'name', 'module', 'moduleName', 'moduleIdentifier'];
-
-const stripBabelLoaderPrefix = log => log.replace(/@?\s*(\.\/~\/babel-loader\/lib\?{[\s\S]*?}!)/g, '');
+const keysToNormalize = [
+	'issuer',
+	'issuerName',
+	'identifier',
+	'name',
+	'module',
+	'moduleName',
+	'moduleIdentifier'
+];
+
+/** Removes all loaders from any resource identifiers found in a string */
+function stripLoaderPrefix(str) {
+	if (typeof str==='string') {
+		return str.replace(/(^|\b|@)(\.\/~|\.{0,2}\/[^\s]+\/node_modules)\/\w+-loader(\/[^?!]+)?(\?\?[\w_.-]+|\?({[\s\S]*?})?)?!/g, '');
+	}
+	return str;
+}
 
-function stripBabelLoaderFromModuleNames(m) {
-	keysToNormalize.forEach(key => {
-		if (key in m) {
-			m[key] = stripBabelLoaderPrefix(m[key]);
+function stripLoaderFromModuleNames(m) {
+	for (let key in m) {
+		if (m.hasOwnProperty(key) && m[key]!=null && ~keysToNormalize.indexOf(key)) {
```

Comment:
Removing the `~` before `keysToNormalize.indexOf(key)` doesn't make a huge difference tho.